src/Cpp/1-getting-started/1-HelloWindow/Input/Mouse.cpp

```diff
@@ -0,0 +1,104 @@
+#include "Mouse.hpp"
+#include "Input.hpp"
+
+#include <GLFW/glfw3.h>
+
+void Mouse::HandleMouseButtonPressed(
+    GLFWwindow* window,
+    const std::int32_t button,
+    const std::int32_t action,
+    const std::int32_t modifiers)
+{
+    _CRT_UNUSED(modifiers);
+    const auto input = static_cast<Input*>(glfwGetWindowUserPointer(window));
```

Comment:
prefer actual type over auto

Reply:
Has been discussed in other comments. otherwise its been updated in this instance